core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -33,13 +33,15 @@ oppia.directive('learnerDashboardIcons', [
         '$scope', 'LearnerDashboardIdsBackendApiService',
         'LearnerDashboardActivityIdsObjectFactory',
         'LearnerPlaylistService',
-        function($scope, LearnerDashboardIdsBackendApiService,
-                 LearnerDashboardActivityIdsObjectFactory,
-                 LearnerPlaylistService) {
-          $scope.activityIsActive = true;
+        function(
+          $scope, LearnerDashboardIdsBackendApiService,
+          LearnerDashboardActivityIdsObjectFactory,
+          LearnerPlaylistService) {
+          $scope.activityIsCurrentlyHoveredOver = true;
 
           $scope.$watch('activityActive', function(value) {
-            $scope.activityIsActive = !$scope.activityIsActive;
+            $scope.activityIsCurrentlyHoveredOver = (
```

Comment:
Can we find some way to "set", and not toggle?

Reply:
Done!